src/models/thelio-mira-b1.0/README.md

```diff
@@ -0,0 +1,89 @@
+        - [NVIDIA GeForce RTX 3080 Ti](https://www.nvidia.com/en-us/geforce/graphics-cards/30-series/rtx-3080-3080ti/) (maximum 1)
+            - HDMI 2.1, 3x DisplayPort 1.4a
+        - [NVIDIA GeForce RTX 3090](https://www.nvidia.com/en-us/geforce/graphics-cards/30-series/rtx-3090/) (maximum 1)
+            - HDMI 2.1, 3x DisplayPort 1.4a
+        - [NVIDIA Quadro RTX 4000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-4000/) (maximum 2)
+            - 3x DisplayPort 1.4, DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA Quadro RTX 5000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-5000/) (maximum 2)
+            - 4x DisplayPort 1.4, DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA Quadro RTX 6000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-6000/) (maximum 2)
+            - 4x DisplayPort 1.4, DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA Quadro RTX 8000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-8000/) (maximum 2)
+            - 4x DisplayPort 1.4, DisplayPort over USB-C (VirtualLink)
+- Expansion
+    - 2x PCIe 4.0/3.0 x16 (GPU slots)
+    - 1x PCIe 4.0/3.0 x16 (additional slot; single-height cards only)
+    - 1x PCIe 3.0 x4
+- Memory
+    - Up to 128GB (4x32GB) quad-channel ECC Unbuffered DDR4 DIMMs @ 3200 MHz
+    - Tested with the following RAM modules (may ship with other tested modules):
+        - [Kingston HyperX HX432C16FB3/8](https://www.kingston.com/dataSheets/HX432C16FB3_8.pdf) (8GB/stick)
+        - [Kingston HyperX HX432C16FB4/16](https://www.kingston.com/dataSheets/HX432C16FB4_16.pdf) (16GB/stick)
+        - [Kingston HyperX HX432C16FB3K4/128](https://www.kingston.com/datasheets/HX432C16FB3K4_128.pdf) (32GB/stick)
+- Networking
+    - 1x 2.5-Gigabit Ethernet (Intel I225-V)
+    - Wi-Fi 6 (Intel AX200)
+- Power
+    - C19 power cable
+    - 650W minimum PSU
+        - Dual-GPU configurations require 750W or 1000W, depending on GPU power requirements
+    - Tested with the following PSU models (may ship with other tested models):
+        - [EVGA 650W B5 80+ Bronze](https://www.evga.com/Products/Specs/PSU.aspx?pn=9C00E82F-02D0-4706-A361-78D42787345B)
+        - [EVGA SuperNOVA 750W G5 80+ Gold](https://www.evga.com/Products/Specs/PSU.aspx?pn=A7699F25-A614-4D88-A339-1DDF3D24615E)
+        - [EVGA SuperNOVA 1000W G5 80+ Gold](https://www.evga.com/products/Specs/PSU.aspx?pn=71d331b7-9306-4218-987f-875fd422fd44)
+        - [EVGA SuperNOVA 1600W T2 80+ Titanium](https://www.evga.com/Products/Specs/PSU.aspx?pn=50804BF2-4D32-4E90-B338-C81CA63AF4E2)
+- Sound
+    - 3.5mm line out, line in, microphone jacks
+    - Up to 7.1-channel audio output
+    - Realtek ALC4080 audio chipset (with Savitech SV3H712 amplifier)
+    - HDMI, DisplayPort, USB-C DisplayPort audio (depending on GPU)
+- Storage
+    - 1x M.2 (PCIe NVMe Gen 4), M key
+        - Backwards compatible with PCIe NVMe Gen 3.
+        - Only active with 11th Gen CPUs.
```

Comment:
I could see this being interpreted as "slot is only active with 11th gen CPUs" instead of "Gen 4 is only active with 11th gen CPUs". Maybe it should be more specific?

Reply:
The slot _is_ only active with 11th Gen CPUs, according to the motherboard manual.
![image](https://user-images.githubusercontent.com/7199422/134421762-1f960c8e-b57f-4d2a-b7fb-9ac27bef707a.png)

Reply:
I originally had the active line above the compatibility line, but switched them because the compatibility related more to the line above than whether it's active or not. I can switch them back if that would clarify this further.

Reply:
Oh ok, I thought it just fell back to 3.0 with a 10th gen CPU. No need to change it, I was just mistaken on the actual functionality of the slot.